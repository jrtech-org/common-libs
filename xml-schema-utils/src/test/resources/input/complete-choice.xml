<?xml version="1.0" encoding="UTF-8"?>
<Document xmlns="urn:bbp:choice:complete">
    <ChoiceTestDocument>
        <AcctId>
            <Prtry>
                <Id />
            </Prtry>
        </AcctId>
        <AcctNm />
        <AcctSvcr>
        </AcctSvcr>
        <OrdrOrgtrElgblty />
        <Charging />
    </ChoiceTestDocument>
</Document>
