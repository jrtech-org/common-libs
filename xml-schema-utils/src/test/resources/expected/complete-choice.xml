<?xml version="1.0" encoding="UTF-8"?>
<Document xmlns="urn:acme:choice:complete">
    <ChoiceTestDocument>
        <AcctId>
            <Prtry>
                <Id />
            </Prtry>
        </AcctId>
        <AcctNm />
        <OwnrId></OwnrId>
        <GrssAmt Ccy="CHF">100</GrssAmt>
        <AcctSvcr>
        </AcctSvcr>
        <OrdrOrgtrElgblty />
        <Charging />
    </ChoiceTestDocument>
</Document>
